site/source/user-manual/service-guides/nextcloud/nextcloud-android.rst

```diff
@@ -44,12 +35,11 @@ Tor Setup
 =========
 You will first need to have the :ref:`Tor daemon running<tor-android>`.
 
-1. First, enable Tor in the Nextcloud Config on Embassy, Services -> Nextcloud -> Config -> Connection Settings -> Connection Type -> LAN and Tor.
-2. On your desktop application, click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
-3. Close the Settings screen and click the account in the top left again, then "Add Account."
-4. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
-5. This will launch your browser and prompt you to log in to your account. Log in and then grant access as we did for LAN.
-6. That's it! You may wish to set up some select folders for remote sync, but for large files, it is best to sync on LAN only, so you can "Skip folders configuration" on the resulting screen if you wish. Check your connection by clicking the newly created account in the client app.
+1. On your desktop application, click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
+2. Close the Settings screen and click the account in the top left again, then "Add Account."
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
```

Comment:
This button actually says "Log in to your Nextcloud" iffin ya care.  I think "Log in" is clear enough though, personally.